_posts/en/newsletters/2023-02-01-newsletter.md

```diff
@@ -0,0 +1,204 @@
+    the secret as a consequence of Bob accepting the first of those
+    payments, the LSP would be able to steal payments for the remainder
+    of the payments to the reused invoice---making the pre-generated
+    invoice solution for HTLCs only secure if Bob trusts his LSP.
+
+    This week, Anthony Towns [proposed][towns async] a solution based on
+    [signature adaptors][topic adaptor signatures].  This would depend
+    on the planned upgrade of LN to use [PTLCs][topic ptlc].  Bob would
+    pre-generate a series of signature nonces and give them to his LSP.
+    The LSP would give a signature nonce to Alice, Alice would choose a
+    message for her proof of payment (e.g. "Alice paid Bob 1000 sats at
+    2023-02-01 12:34:56Z"), and then use Bob's nonce and her message
+    to generate a signature adaptor for her PTLC.  When Bob comes back
+    online, the LSP forwards him the payment and Bob verifies the nonce
+    hasn't been used before, that he agrees with the message, that the
+    payment is otherwise valid, and that the signature adaptor math is
+    valid; he then accepts the payment and Alice, when she ultimately
+    receives the settled PTLC, will have a signature from Bob that
+    commits to her chosen message.
+
+    Towns's solution involves the LSP receiving pre-generated invoices
+    from Bob---this is similar to the insecure/trusted solution for
+    HTLCs, yet the PTLC signature adaptor solution is secure and
+    trustless because each payment from a different spender (like Alice)
+    uses a different PTLC public key point and Bob is able to prevent
+    nonce reuse.  Each PTLC point is different because it derives from a
+    unique message selected by each spender.  Bob is able to prevent
+    nonce reuse by checking for reused nonces before he accepts each
+    payment.
+
+    In his post, Towns [references][towns sa1] two [previous][towns sa2]
+    mailing list posts he's written about LN proof of payment using
+    signature adaptors.  As of this writing, no replies to the idea have
+    been posted to the mailing list.
+
+<!-- FIXME: harding to add releases/RCs if any -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #26471][] reduces the default mempool capacity to 5MB
+  (from 300MB) when a user turns on `-blocksonly` mode. Since unused
+  mempool memory is shared with dbcache, this change also reduces the
+  default dbcache size in `-blocksonly` mode. Users may still configure
+  a larger mempool capacity using the `-maxmempool` option.
```

Comment:
```suggestion
  a larger mempool capacity using the `-maxmempool` configuration option.
```

Reply:
Thanks, but didn't take because "configure using the configuration option" seems a bit redundant to me